Sources/ComposableArchitecture/Effects/Cancellation.swift

```diff
@@ -182,7 +182,7 @@ extension Task where Success == Never, Failure == Never {
   /// Cancel any currently in-flight operation with the given identifier.
   ///
   /// - Parameter id: An identifier.
-  public static func cancel<ID: Hashable & Sendable>(id: ID) {
+  public static func cancel<ID: Hashable>(id: ID) {
```

Comment:
This change is mainly to be consistent. I also tried to mark _everything_ `Hashable & Sendable` but it bled out and introduced warnings to existing code, so I held off on that for now (we could explore it again in another PR).